client/src/theme/Components/_index.scss

```diff
@@ -5,6 +5,7 @@
 @import './loading-bar';
 @import './search-menu';
 @import './share-nav-primary';
+@import './toggle_primary';
```

Comment:
I moved this to the backend components folder. Even though it's currently used on the front-end, it's used in the context of "backend" styles.